app/assets/javascripts/admin/modules/ga4-button-setup.js

```diff
@@ -0,0 +1,27 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+;(function (Modules) {
+  function Ga4ButtonSetup(module) {
+    this.module = module
+  }
+
+  Ga4ButtonSetup.prototype.init = function () {
+    const buttons = this.module.querySelectorAll('button')
```

Comment:
Do we have to worry about [links with a role of 'button'](https://design-system.service.gov.uk/components/button/#start-buttons) too?

Reply:
Good point, it's not something that we see on this page, but I've adjusted the query selector and added a test case for it